packages/unchained-client/src/evm/parser/index.ts

```diff
@@ -14,19 +16,23 @@ export * from './utils'
 export interface TransactionParserArgs {
   chainId: ChainId
   assetId: AssetId
+  api: Api
   rpcUrl: string
 }
 
 export class BaseTransactionParser<T extends Tx> {
   chainId: ChainId
   assetId: AssetId
 
+  protected readonly api: Api
```

Comment:
add unchained api client class property. this was needed to query nft token metadata as csp rules were problematic when querying arbitrary metadata endpoints from within the client directly.